src/lib/logger.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
```

Comment:
I think we might need to add this stream into `lib/httpd.js`, rather than here. The reason is that if somehow there's a flow through the application where the restify server is invoked, but this file isn't, we won't be attaching this stream. We might have got away with this because `logger.js` is used in the stack _somewhere_.

You could create a new file for this stream function, import it into `httpd.js` and add the stream in there instead if we want to ensure that we get it for every log instance.

Reply:
Agreed, and resolved